### PR TITLE
CCN3-1903: Add question about whether the PIH are known to the local council

### PR DESCRIPTION
**Pull request by smjiko**
## Description

Adds a question to the health check after the summary page but before declaration about being known to the council

## Todo's before PR

- [x] Branch has been rebased against develop
- [x] Unit tests passed (`make test`)
- [x] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [x] Templates have been checked against the relevant user-story

## Migrations

- [x] Null fields in models specifies default value
- [x] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assignees (those who'll merge)
- [x] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [x] Code syntax formatting checked
- [x] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [x] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

**Comment by LRydeard** [comment]
rerun jenkins build

File: application/views/other_people_health_check/local_authorities.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    success_url = 'Health-Check-Declaration'
+
+    def get_initial(self):
+        """
+        Get initial defines the initial data for the form instance that is to be rendered on the page
+        :return: a dictionary mapping form field names, to values of the correct type
+        """
+        initial = super().get_initial()
+
+        try:
```
Comment (LRydeard) [inline application/views/other_people_health_check/local_authorities.py]: This try except should not be necessary this far into a health check.

File: application/views/other_people_health_check/local_authorities.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            person_id = self.request.GET['person_id']
+            person_record = AdultInHome.objects.get(pk=person_id)
+            initial['known_to_council'] = person_record.current_treatment
```
Comment (LRydeard) [inline application/views/other_people_health_check/local_authorities.py]: The wrong person_record field is being looked at here.

File: application/views/other_people_health_check/summary.py [hunk 3]
```diff
@@ -12,7 +12,8 @@ def get_context_data(self, **kwargs):
 
         context['person_id'] = self.request.GET.get('person_id')
         person_object = AdultInHome.objects.get(pk=self.request.GET.get('person_id'))
-        context['declaration_url'] = build_url('Health-Check-Declaration', get={'person_id': context['person_id']})
+        context['localauthorities_url'] = build_url('Health-Check-Local-Authorities', get={'person_id': context['person_id']})
+        #context['declaration_url'] = build_url('Health-Check-Declaration', get={'person_id': context['person_id']})
```
Comment (LRydeard) [inline application/views/other_people_health_check/summary.py]: Is this comment left in for a reason?

**Comment by smjiko** [comment]
rerun jenkins build

**Comment by LRydeard** [comment]
rerun jenkins build

**Pull request merged**